pkg/routers/api_v1.go

```diff
@@ -66,13 +68,23 @@ func (d defaultHandlers) override(w http.ResponseWriter, r *http.Request) {
 	handlers.Override(w, r)
 }
 
+type apiOAuthHandler interface {
+	CreateAPIAccessToken(w http.ResponseWriter, r *http.Request)
+}
+
+type apiOAuthMiddleware interface {
+	AuthorizeAPI(next http.Handler) http.Handler
+}
+
```

Comment:
FWIW, I'm planning on removing these interfaces. I think they are getting unruly and not adding much benefit. I think this is OK for now, but just a heads up that a refactor is on the horizon :)